src/vmaware.hpp

```diff
@@ -1747,24 +1750,27 @@ struct VM {
      * @category Windows
      */
     [[nodiscard]] static bool sunbelt_check() try {
-        #if (!MSVC)
+    #if (!MSVC)
             return false;
-        #else
+    #else
             if (disabled(SUNBELT)) {
-                #ifdef __VMAWARE_DEBUG__
-                    debug("SUNBELT: ", "precondition return called");
-                #endif
+    #ifdef __VMAWARE_DEBUG__
+                debug("SUNBELT: ", "precondition return called");
+    #endif
                 return false;
             }
 
-            return (exists("C:\\analysis"));
-        #endif
-    } catch (...) { 
-        #ifdef __VMAWARE_DEBUG__
+            // Use wide string literal
+            return exists(L"C:\\analysis");
+    #endif
+        }
+        catch (...) {
+    #ifdef __VMAWARE_DEBUG__
             debug("SUNBELT: catched error, returned false");
-        #endif
-        return false;
-    }
+    #endif
+            return false;
+        }
+
```

Comment:
I don't really see the point of the indentation, but it really doesn't matter. I'll update that in a future commit, don't worry about it.